front/src/views/movies/movies.view.html

```diff
@@ -1,3 +1,27 @@
-<div>    
-  This is a movie page.
-</div>
+<div class="content">
+  <el-row class="el-row-wrap" type="flex" justify="center">
+    <el-col :span="6" v-for="movie in getMoviesList" class="grid-col-wrap" :key="movie.id">
+      <div class="grid-content-wrap">
+        <el-card :body-style="{ padding: '0px' }">
+          <el-image fit='cover' :src="getImgSrc(movie.imageSource)"></el-image>
+          <div class="grid-content-main">
+            <div class="movie-title">
+              <span>{{movie.title}}</span>
+            </div>
+            <el-row type="flex" justify="space-between">
+              <el-col :span="24">
+                <el-rate class="rate" v-model="movie.rate" disabled
+                  :colors="['var(--ff-primary-color)', 'var(--ff-primary-color)', 'var(--ff-primary-color)']"></el-rate>
+              </el-col>
+              <el-col :span="6"class="releaseYear">{{movie.releaseYear}}</el-col>
+            </el-row>
+            <div class="actors-list">
+              <span>{{movie.actorsList | firstElement}}</span>
```

Comment:
firstElement doesn't have sense - you can use `<span>{{movie.actorsList[0]}}, </span>`